.github/workflows/maven.yaml

```diff
@@ -46,12 +46,24 @@ jobs:
       name: 'Test - Unit'
       run: './mvnw -B test'
     - id: 'step-6'
+      name: 'JDK 17 - only for Sonar'
```

Comment:
This file is generated via the kts file in `.github/kts/maven.main.kts. I would not change it directly :D

Switch to j17 is also done on the 0.7/sb3 branch and should be merged soon, so I would recommend to not merge this PR sadly :(

Reply:
all good, just playing around, no not merge! :-)

Reply:
Somehow converted itself into a play-around branch... its current content is of no value!